src/wing/CCPACSWingProfilePointList.cpp

```diff
@@ -196,6 +198,49 @@ void CCPACSWingProfilePointList::ReadCPACS(TixiDocumentHandle tixiHandle)
     }
 }
 
+void CCPACSWingProfilePointList::WriteCPACS(TixiDocumentHandle tixiHandle, const std::string& profileXPath)
+{
+    std::string path;
+    std::string elementPath;
+        
+    // Set the element "point"
+    TixiSaveExt::TixiSaveElement(tixiHandle, profileXPath.c_str(), "pointList");
+    
+    path = profileXPath + "/pointList";
+    // TODO : find how to get the number of 'point' (and eventually the definition...)
```

Comment:
The same issue here (as for the fuselage profile). I don't see the meaning of multiple points. Or is it, to support older cpacs datasets with numerous `<point/>` entries instead of a `<pointList/>`?